handlers/network_data_aggregator.py

```diff
@@ -0,0 +1,12 @@
+import redis
+import json
+
+redis_client = redis.Redis(host='localhost', port=6379, db=0)
```

Comment:
Maybe we should move host and port to a config file? You run this line in many places